src/helpers.ts

```diff
@@ -257,3 +257,50 @@ export const CACHE_PATH = join(cwd, '.synthetics', process.pid.toString());
 export function getDurationInUs(duration: number) {
   return Math.trunc(duration * 1e6);
 }
+
+export function megabytesToBytes(megabytes: number) {
+  return megabytes * 1024 * 1024;
+}
+
+export const networkConditionDefaults = {
```

Comment:
nit: DEFAULT_NETWORK_CONDITIONS capital case to reason about that its a constant